apps/portals/adknowledgeportal/src/config/routesConfig.tsx

```diff
@@ -122,11 +122,12 @@ const routes: RouteObject[] = [
           ),
       },
       {
-        path: 'Contribute/FormSubmission',
-        lazy: () =>
-          import('@/pages/Contribute/Contribute-FormSubmission').then(
-            convertModuleToRouteObject,
-          ),
+        // The FormSubmission page has been replaced with a link to Service Desk
+        // path: 'Contribute/FormSubmission',
+        // lazy: () =>
+        //   import('@/pages/Contribute/Contribute-FormSubmission').then(
+        //     convertModuleToRouteObject,
+        //   ),
```

Comment:
I thought I had changed this in an earlier PR, but somehow it ended up in this one...

This page is not exposed anywhere in the portal and I checked with Ann C that it's ok to hide this.